### PR TITLE
Create distributed-systems.md

### LINKED ISSUE #1300: [hi] Localize `Distributed System` in Hindi
- https://glossary.cncf.io/distributed-systems/

### PR DESCRIPTION
**Pull request by Aryan-Singla**
### Describe your changes
Localize Distributed System in Hindi in hindi

### Related issue number or link (ex: `resolves #issue-number`)
resolves #1300 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/hi/distributed-systems.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: डिस्ट्रिब्यूटेड सिस्टम (Distributed System)
+status: Completed
+category: अवधारणा
+tags: ["आर्किटेक्चर"]
+---
+
+डिस्ट्रिब्यूटेड सिस्टम स्वायत्त कंप्यूटिंग तत्वों का एक संग्रह है
+एक नेटवर्क पर जुड़ा हुआ है जो उपयोगकर्ताओं को एकल सुसंगत प्रणाली के रूप में दिखाई देता है।
```
Comment (aj11anuj) [inline content/hi/distributed-systems.md:9]:
```suggestion
जो एक नेटवर्क पर जुड़ा हुआ है और उपयोगकर्ताओं को एकल सुसंगत प्रणाली के रूप में दिखाई देता है।
```

File: content/hi/distributed-systems.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+आम तौर पर [नोड्स](/nodes/) के रूप में संदर्भित, ये घटक हार्डवेयर डिवाइस (जैसे कंप्यूटर, मोबाइल फोन) या सॉफ्टवेयर प्रक्रियाएं हो सकते हैं।
```
Comment (aj11anuj) [inline content/hi/distributed-systems.md:10]:
```suggestion
आम तौर पर [नोड्स](/nodes/) के रूप में संदर्भित, ये भाग हार्डवेयर डिवाइस जैसे - कंप्यूटर, मोबाइल फोन या सॉफ्टवेयर प्रक्रियाएं हो सकते हैं।
```

File: content/hi/distributed-systems.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+वितरित कंप्यूटिंग के बिना, आज जिन कई अनुप्रयोगों पर हम भरोसा करते हैं, वे संभव नहीं होंगे।
+
+परंपरागत रूप से, सिस्टम लंबवत रूप से [स्केल](/scalability/) होते हैं।
+तभी आप किसी व्यक्तिगत मशीन में अधिक सीपीयू या मेमोरी जोड़ते हैं।
+वर्टिकल स्केलिंग में समय लगता है, इसके लिए डाउनटाइम की आवश्यकता होती है और यह जल्दी ही अपनी सीमा तक पहुंच जाती है।
+
+## समाधान
+
+डिस्ट्रिब्यूटेड सिस्टम [होरिजेंटल स्केलिंग](/horizontal-scaling/) की अनुमति देती हैं (उदाहरण के लिए जब भी जरूरत हो सिस्टम में अधिक नोड्स जोड़ना)।
+इसे स्वचालित किया जा सकता है जिससे सिस्टम कार्यभार या संसाधन खपत में अचानक वृद्धि को संभाल सकता है।
```
Comment (aj11anuj) [inline content/hi/distributed-systems.md:27]:
```suggestion
इसे स्वचालित किया जा सकता है जिससे सिस्टम के कार्यभार या संसाधन की खपत में अचानक हुई वृद्धि को संभालना सरल हो सके।
```

File: content/hi/distributed-systems.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+एक गैर-वितरित प्रणाली विफलता के जोखिमों को उजागर करती है क्योंकि यदि एक मशीन विफल हो जाती है, तो पूरी प्रणाली विफल हो जाती है।
+एक वितरित प्रणाली को इस तरह से डिज़ाइन किया जा सकता है कि,
+भले ही कुछ मशीनें खराब हो जाएं, फिर भी समग्र प्रणाली समान परिणाम देने के लिए काम करना जारी रख सकती है।
```
Comment (aj11anuj) [inline content/hi/distributed-systems.md:31]:
```suggestion
भले ही कुछ मशीनें खराब हो जाएं, फिर भी समग्र प्रणाली समान परिणाम देने के लिए काम करना जारी रख सके।
```

**Review by kumarankit999 — COMMENTED** [review]
Otherwise it looks good!

**Comment by Aryan-Singla** [comment]
Hello @aj11anuj if you have bandwidth then Please check this localization is there any need for improvement

File: content/hi/distributed-systems.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+डिस्ट्रिब्यूटेड सिस्टम स्वायत्त कंप्यूटिंग तत्वों का एक संग्रह है
+जो एक नेटवर्क पर जुड़ा हुआ है और उपयोगकर्ताओं को एकल सुसंगत प्रणाली के रूप में दिखाई देता है।
+आम तौर पर [नोड्स](/nodes/) के रूप में संदर्भित, ये भाग हार्डवेयर डिवाइस जैसे - कंप्यूटर, मोबाइल फोन या सॉफ्टवेयर प्रक्रियाएं हो सकते हैं।
+नोड्स को एक सामान्य लक्ष्य प्राप्त करने के लिए प्रोग्राम किया जाता है और सहयोग करने के लिए, वे नेटवर्क पर संदेशों का आदान-प्रदान करते हैं।
+
+## यह समस्या का समाधान करता है
+
+आज अनेक आधुनिक अनुप्रयोग इतने बड़े हो गए हैं कि उन्हें संचालित करने के लिए सुपर कंप्यूटर की आवश्यकता होगी।
+जीमेल या नेटफ्लिक्स के बारे में सोचें। कोई भी कंप्यूटर संपूर्ण एप्लिकेशन को होस्ट करने के लिए पर्याप्त शक्तिशाली नहीं है।
```
Comment (aj11anuj) [inline content/hi/distributed-systems.md:16]:
```suggestion
जी-मेल या नेटफ्लिक्स के बारे में सोचें। कोई भी कंप्यूटर संपूर्ण एप्लिकेशन को होस्ट करने के लिए पर्याप्त शक्तिशाली नहीं है।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Replace 'वितरित', 'गैर-वितरित', 'प्रणाली', 'सुसंगत', 'स्वायत्त' words with relevant alternative, wherever you have used them.

**Comment by Aryan-Singla** [comment]
> Replace 'वितरित', 'गैर-वितरित', 'प्रणाली', 'सुसंगत', 'स्वायत्त' words with relevant alternative, wherever you have used them.
Update all the items with other words

File: content/hi/distributed-systems.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+जो एक नेटवर्क पर जुड़ा हुआ है और उपयोगकर्ताओं को एकल स्पष्ट सिस्टम के रूप में दिखाई देता है।
+आम तौर पर [नोड्स](/nodes/) के रूप में संदर्भित, ये भाग हार्डवेयर डिवाइस जैसे - कंप्यूटर, मोबाइल फोन या सॉफ्टवेयर प्रक्रियाएं हो सकते हैं।
+नोड्स को एक सामान्य लक्ष्य प्राप्त करने के लिए प्रोग्राम किया जाता है और सहयोग करने के लिए, वे नेटवर्क पर संदेशों का आदान-प्रदान करते हैं।
+
+## यह समस्या का समाधान करता है
+
+आज अनेक आधुनिक अनुप्रयोग इतने बड़े हो गए हैं कि उन्हें संचालित करने के लिए सुपर कंप्यूटर की आवश्यकता होगी।
+जी-मेल या नेटफ्लिक्स के बारे में सोचें। कोई भी कंप्यूटर संपूर्ण एप्लिकेशन को होस्ट करने के लिए पर्याप्त शक्तिशाली नहीं है।
+कई कंप्यूटरों को जोड़ने से, गणना शक्ति लगभग असीमित हो जाती है।
+डिसटीब्यूट कंप्यूटिंग के बिना, आज जिन कई अनुप्रयोगों पर हम भरोसा करते हैं, वे संभव नहीं होंगे।
```
Comment (aj11anuj) [inline content/hi/distributed-systems.md:18]:
```suggestion
डिस्ट्रिब्यूटेड कंप्यूटिंग के बिना, आज जिन कई अनुप्रयोगों पर हम भरोसा करते हैं, वे संभव नहीं होंगे।
```

File: content/hi/distributed-systems.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+परंपरागत रूप से, सिस्टम लंबवत रूप से [स्केल](/scalability/) होते हैं।
+तभी आप किसी व्यक्तिगत मशीन में अधिक सीपीयू या मेमोरी जोड़ते हैं।
```
Comment (aj11anuj) [inline content/hi/distributed-systems.md:21]:
```suggestion
तभी आप किसी व्यक्तिगत मशीन में अधिक सीपीयू या मेमोरी जोड़ पाते हैं।
```

File: content/hi/distributed-systems.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+वर्टिकल स्केलिंग में समय लगता है, इसके लिए डाउनटाइम की आवश्यकता होती है और यह जल्दी ही अपनी सीमा तक पहुंच जाती है।
+
+## समाधान
+
+डिस्ट्रिब्यूटेड सिस्टम [होरिजेंटल स्केलिंग](/horizontal-scaling/) की अनुमति देती हैं (उदाहरण के लिए जब भी जरूरत हो सिस्टम में अधिक नोड्स जोड़ना)।
```
Comment (aj11anuj) [inline content/hi/distributed-systems.md:26]:
```suggestion
डिस्ट्रिब्यूटेड सिस्टम [होरिजेंटल स्केलिंग](/horizontal-scaling/) की अनुमति देती हैं (उदाहरण के लिए जरूरत के समय सिस्टम में अधिक नोड्स जोड़ना)।
```

File: content/hi/distributed-systems.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+इसे ऑटोमेटिक किया जा सकता है जिससे सिस्टम के कार्यभार या संसाधन की खपत में अचानक हुई वृद्धि को संभालना सरल हो सके।
+
+एक नॉन डिसटीब्यूट सिस्टम विफलता के जोखिमों को उजागर करती है क्योंकि यदि एक मशीन विफल हो जाती है, तो पूरी सिस्टम विफल हो जाती है।
```
Comment (aj11anuj) [inline content/hi/distributed-systems.md:29]:
```suggestion
एक नॉन-डिस्ट्रिब्यूटेड सिस्टम विफलता के जोखिमों को उजागर करती है क्योंकि यदि एक मशीन विफल हो जाती है, तो पुरा सिस्टम विफल हो जाता है।
```

File: content/hi/distributed-systems.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+एक डिसटीब्यूट सिस्टम को इस तरह से डिज़ाइन किया जा सकता है कि,
```
Comment (aj11anuj) [inline content/hi/distributed-systems.md:30]:
```suggestion
एक डिस्ट्रिब्यूटेड सिस्टम को इस तरह से डिज़ाइन किया जा सकता है कि,
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
.

**Review by aj11anuj — APPROVED** [review]
Lgtm

**Comment by bishal7679** [comment]
@Aryan-Singla Pls keep patience!
Will review this PR for final soon

**Comment by Aryan-Singla** [comment]
@bishal7679 Yes bishal I have no hurry

**Comment by Aryan-Singla** [comment]
@bishal7679 Please check PR pending from long time

**Comment by Aryan-Singla** [comment]
@bishal7679 Also check this

**Comment by Aryan-Singla** [comment]
Hello @kumarankit999, @aj11anuj, @abhay-raj19. I think @bishal7679 is busy nowadays. Can you please check and merge this PR? It's been pending for a long time, and we've all invested time in this localization.

**Review by abhay-raj19 — APPROVED** [review]
Looks Good to me !!
Would love to see your contribution to Glossary !

File: content/hi/distributed-systems.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+category: अवधारणा
+tags: ["आर्किटेक्चर"]
+---
+
+डिस्ट्रिब्यूटेड सिस्टम ऑटोनॉमस कंप्यूटिंग तत्वों का एक कलेक्शन है
+जो एक नेटवर्क पर जुड़ा हुआ है और उपयोगकर्ताओं को एकल स्पष्ट सिस्टम के रूप में दिखाई देता है।
+आम तौर पर [नोड्स](/nodes/) के रूप में संदर्भित, ये भाग हार्डवेयर डिवाइस जैसे - कंप्यूटर, मोबाइल फोन या सॉफ्टवेयर प्रक्रियाएं हो सकते हैं।
+नोड्स को एक सामान्य लक्ष्य प्राप्त करने के लिए प्रोग्राम किया जाता है और सहयोग करने के लिए, वे नेटवर्क पर संदेशों का आदान-प्रदान करते हैं।
+
+## यह समस्या का समाधान करता है
```
Comment (bishal7679) [inline content/hi/distributed-systems.md:13]:
```suggestion
## समस्या
```

**Comment by bishal7679** [comment]
Delete the lines from 33 - 35

**Comment by Aryan-Singla** [comment]
@bishal7679 Check now

File: content/hi/distributed-systems.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+वर्टिकल स्केलिंग में समय लगता है, इसके लिए डाउनटाइम की आवश्यकता होती है और यह जल्दी ही अपनी सीमा तक पहुंच जाती है।
+
+## समाधान
+
+डिस्ट्रिब्यूटेड सिस्टम [होरिजेंटल स्केलिंग](/horizontal-scaling/) की अनुमति देती हैं (उदाहरण के लिए जरूरत के समय सिस्टम में अधिक नोड्स जोड़ना)।
+इसे ऑटोमेटिक किया जा सकता है जिससे सिस्टम के कार्यभार या संसाधन की खपत में अचानक हुई वृद्धि को संभालना सरल हो सके।
+
+एक नॉन-डिस्ट्रिब्यूटेड सिस्टम विफलता के जोखिमों को उजागर करती है क्योंकि यदि एक मशीन विफल हो जाती है, तो पुरा सिस्टम विफल हो जाता है।
+एक डिस्ट्रिब्यूटेड सिस्टम को इस तरह से डिज़ाइन किया जा सकता है कि,
+भले ही कुछ मशीनें खराब हो जाएं, फिर भी समग्र सिस्टम समान परिणाम देने के लिए काम करना जारी रख सके।
```
Comment (bishal7679) [inline content/hi/distributed-systems.md:31]:
```suggestion
भले ही कुछ मशीनें खराब हो जाएं, फिर भी समग्र सिस्टम समान परिणाम देने के लिए काम करना जारी रख सके।

```

**Comment by abhay-raj19** [comment]
Thanks for contributing to Glossary @Aryan-Singla !!

**Pull request merged**